src/components/FileInput.js

```diff
@@ -68,6 +70,19 @@ class FileInput extends Component {
       this.input.click();
     }
   }
+
+  getImageThumbnail(file = { mimeType: '' }) {
+    const reader = new FileReader();
+    if (file) {
+      if (file.mimeType.match('image.*')) {
+        reader.readAsDataURL(file);
+      } else {
+        this.setState({ image: '' });
```

Comment:
The initial state should have its `image` key (to null) so we always have the state defined.